### ISSUE TITLE
Font types of placeholder and content don't match

### ISSUE THREAD

**Issue by bernhardengl** [raulriera/TextFieldEffects#199]
Tried Hoshi. Font set to "System" in Interface Builder. Content indeed seems to match the System Font but the Placeholder text clearly isn't. Didn't find a way to fix that.

**Comment by TheNosplay** [comment]
I am getting this error:
```
2020-03-09 10:59:32.039959+0100 CoreText note: Client requested name ".SFUI-Regular", it will get TimesNewRomanPSMT rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[UIFont systemFontOfSize:].
2020-03-09 10:59:32.040177+0100  CoreText note: Set a breakpoint on CTFontLogSystemFontNameRequest to debug.
2020-03-09 10:59:32.041455+0100 CoreText note: Client requested name ".SFUI-Regular", it will get TimesNewRomanPSMT rather than the intended font. All system UI font access should be through proper APIs such as CTFontCreateUIFontForLanguage() or +[UIFont systemFontOfSize:].
```

**Comment by ZacharyKhan** [comment]
I am also experiencing this same issue and I get the same error as @TheNosplay. I tried to edit `private func placeholderFontFromFont(_ font: UIFont) -> UIFont!` too, but still getting this error.

**Comment by ZacharyKhan** [comment]
Fixed by manually implementing #191 - see the changes in files. It seems the pod trunk has not been updated with the latest PR's.
